src/javascript/app/pages/user/view_popup/view_popup.js

```diff
@@ -250,6 +253,10 @@ const ViewPopup = (() => {
             $container.find('#errMsg').setVisibility(0);
         }
 
+        const { barrier, contract_type, entry_spot } = contract;
+        if (Reset.isReset(contract_type) && (entry_spot !== barrier)) {
```

Comment:
do we need a `+` to make sure entry spot and barrier are both integers? 😅 forever suspicious of API formats